Core/include/Acts/Geometry/GeometryIdentifier.hpp

```diff
@@ -95,6 +101,7 @@ class GeometryIdentifier {
   static constexpr Value kLayerMask     = 0x0000fff000000000;
   /// (2^8)-1 = 255 approach surfaces
   static constexpr Value kApproachMask  = 0x0000000ff0000000;
+  static constexpr Value kPassiveMask   = kApproachMask;
```

Comment:
i.e. an alias? Maybe renaming it altogether is the better approach (pun intended).

Reply:
For the moment (as long as we have approach surfaces in the TrackingGeometry) they are handy to be named such, because it helps in the debugging, The `Detector` model doesn't need that concept anymore.